README.md

```diff
@@ -127,6 +127,32 @@ private async Task<ulong> WriteToDatabaseAsync(PostgreSQLCopyHelper<TestEntity>
 }
 ```
 
+## PostgreSQLCopyHelper.NodaTime: Noda Time Support ##
```

Comment:
```suggestion
## PostgreSQLCopyHelper.NodaTime: NodaTime Support ##
```

Reply:
I feel like this section should go below the `Case-Sensitive Identifiers` section as it is "more related" to the core that NodaTime support.